packages/eslint-plugin-pf-codemods/test/testHelpers.js

```diff
@@ -434,34 +428,67 @@ function getMoveSpecifiersValidtests(specifiersToMoveArray) {
     tests.push({
       code: `import { ${specifier} } from 'foo';`,
     });
-    // Export
-    tests.push({
-      code: `export { ${specifier} as CustomExport } from '@patternfly/react-${package}/deprecated';`,
-    });
-    // Export from dist
-    tests.push({
-      code: `export { ${specifier} as CustomExport } from '@patternfly/react-${package}/dist/esm/deprecated/components/${specifier}/index.js';`,
-    });
     // No @patternfly/react-core export
     tests.push({
       code: `export { ${specifier} as CustomExport } from 'foo';`,
     });
+
+    if (movingFrom === "deprecated") {
+      tests.push({
+        code: `import { ${specifier} } from '@patternfly/react-${package}/next';`,
```

Comment:
I very likely am just missing something obvious here, but why are you testing the next subDir if it's being moved from deprecated?

Reply:
That was added a bit ago, but I think it was just to check that something from the /next directory wasn't going to be deprecated. Probably not necessary so I don't mind removing it if we don't really need it.

Reply:
Ah ok, that makes sense. I'm not opposed to keeping it.